sdks/python/apache_beam/yaml/yaml_transform.py

```diff
@@ -963,10 +963,13 @@ def expand(self, pcolls):
         self._spec['input'] = {name: name for name in pcolls.keys()}
     python_provider = yaml_provider.InlineProvider({})
 
+    # Label goog-dataflow-yaml if job is started using Beam YAML.
     options = pipeline.options.view_as(GoogleCloudOptions)
+    yaml_version = (
+        'goog-dataflow-yaml=' + beam.version.__version__.replace('.', '_'))
     if not options.labels:
       options.labels = []
-    options.labels += ["yaml=true"]
+    options.labels.append(yaml_version)
```

Comment:
Will this result in this label getting appended multiple times if it's already there?